pkg/apis/verify/v1beta1/metadata_types.go

```diff
@@ -23,16 +23,18 @@ import (
 // SigningSpec provides the details for the metadata generator and signer
 // +k8s:openapi-gen=true
 type MetadataSigningSpec struct {
-	EntityID         string `json:"entityID,omitempty"`
-	PostURL          string `json:"postURL,omitempty"`
-	RedirectURL      string `json:"redirectURL,omitempty"`
-	OrgName          string `json:"orgName,omitempty"`
-	OrgDisplayName   string `json:"orgDisplayName,omitempty"`
-	OrgURL           string `json:"orgURL,omitempty"`
-	ContactCompany   string `json:"contactCompany,omitempty"`
-	ContactGivenName string `json:"contactGivenName,omitempty"`
-	ContactSurname   string `json:"contactSurname,omitempty"`
-	ContactEmail     string `json:"contactEmail,omitempty"`
+	EntityID              string `json:"entityID,omitempty"`
+	PostURL               string `json:"postURL,omitempty"`
+	RedirectURL           string `json:"redirectURL,omitempty"`
+	OrgName               string `json:"orgName,omitempty"`
+	OrgDisplayName        string `json:"orgDisplayName,omitempty"`
+	OrgURL                string `json:"orgURL,omitempty"`
+	ContactCompany        string `json:"contactCompany,omitempty"`
+	ContactGivenName      string `json:"contactGivenName,omitempty"`
+	ContactSurname        string `json:"contactSurname,omitempty"`
+	ContactEmail          string `json:"contactEmail,omitempty"`
+	SigningCertificate    string `json:"signingCertificate,omitempty"`
```

Comment:
Similarly here, would SamlSigningCertificate and SamlEncryptionCertificate be clearer?

Reply:
Good shout, I'll change it. The two places you've mentioned above are actually autogenerated from them being specified here.